src/hooks/usePantry.ts

```diff
@@ -65,19 +66,26 @@ export default function usePantry() {
     const project = isString(input) ? input : input.project
 
     const yaml = (() => {
-      for (const prefix of pantry_paths()) {
-        if (!prefix.exists()) throw new PantryNotFoundError(prefix.parent())
-        const dir = prefix.join(project)
-        const filename = dir.join("package.yml")
-        if (!filename.exists()) continue
+      let memo: Promise<PlainObject>
 
-        let memo: Promise<PlainObject> | undefined
+      const parseYAMLFile = (filename: Path) => {
+        if (!filename.exists()) return
 
-        return () => memo ?? (memo = filename.readYAML()
+        return filename.readYAML()
           .then(validate.obj)
-          .catch(cause => { throw new PantryParseError(project, filename, cause) }))
+          .catch(cause => { throw new PantryParseError(project, filename, cause) })
       }
-      throw new PackageNotFoundError(project)
+
+      return () => memo ?? (memo = (async () => {
+        const pkg = await find(project)
```

Comment:
doesn't this mean our `project` now in fact returns display names and provides matches?

Reply:
Yes, defaults to all matchers, so it'll match `agpt.co`, `Auto-GPT` or `auto-gpt`, by default.

Reply:
Or like I could probably whip this up, you have enough going on

Reply:
you're welcome to jump on it, if you wanted. it was a weekend whim, since we were adding display names i figured i'd help the users out.

Reply:
yeah cool. I wanted this feature anyway for a bunch of places in brewkit and tea/cli